### PR TITLE
Returning number of transactions pending in block not queue

### PR DESCRIPTION
**Pull request by tomusdrw**

### CODE REVIEW

**Comment by arkpar** [comment]
`block_pending` name is a little non descriptive, since it is a number of transactions. Looks good otherwise

**Comment by tomusdrw** [comment]
Updated.

File: rpc/src/v1/impls/eth.rs [hunk 1]
```diff
@@ -236,7 +236,9 @@ impl<C, S, A, M, EM> Eth for EthClient<C, S, A, M, EM>
 	fn block_transaction_count_by_number(&self, params: Params) -> Result<Value, Error> {
 		from_params::<(BlockNumber,)>(params)
 			.and_then(|(block_number,)| match block_number {
-				BlockNumber::Pending => to_value(&U256::from(take_weak!(self.miner).status().transaction_queue_pending)),
+				BlockNumber::Pending =>to_value(
```
Comment (gavofyork) [inline rpc/src/v1/impls/eth.rs]: missing space

Comment (tomusdrw) [inline rpc/src/v1/impls/eth.rs]: fixed

**Pull request merged**